src/Files.App/Views/Settings/AboutPage.xaml.cs

```diff
@@ -17,5 +20,17 @@ public AboutPage()
 
 			ViewModel = new AboutViewModel();
 		}
+
+		private void ThirdPartyLicenses_Click(object sender, bool e)
+		{
+			if (e && ViewModel.ThirdPartyNotices is null)
+				ViewModel.LoadThirdPartyNotices();
+		}
```

Comment:
```suggestion
		private void ThirdPartyLicenses_Click(object sender, RoutedEventArgs args)
		{
			if (ViewModel.ThirdPartyNotices is null)
				ViewModel.LoadThirdPartyNotices();
		}

```

Reply:
`bool` is the correct event argument here. `RoutedEventArgs` would result in a compile error.

Reply:
Opps, you right :)